modules/ossm-tutorial-bookinfo-removing.adoc

```diff
@@ -6,37 +6,49 @@ This TASK module included in the following assemblies:
 [id="ossm-tutorial-bookinfo-removing_{context}"]
 = Removing the Bookinfo application
 
-When you finish with the Bookinfo application, you can remove it by running the cleanup script.
+Follow these steps to remove the Bookinfo application.
+
+[id="ossm-delete-bookinfo-project_{context}"]
+== Delete the bookinfo project
 
-[TIP]
-====
-Several of the other {ProductShortName} tutorials also use the Bookinfo application.  Do not run the cleanup script if you plan to explore other {ProductShortName} tutorials.
-====
 .Procedure
 
-. Download the cleanup script:
-+
+. Log into the {product-title} web console.
 
-----
-$ curl -o cleanup.sh https://raw.githubusercontent.com/Maistra/bookinfo/master/cleanup.sh && chmod +x ./cleanup.sh
-----
+. Click to *Home* -> *Projects*.
 
-. Delete the Bookinfo virtualservice, gateway, and terminate the Pods by running the cleanup script:
-+
+. Click on the `bookinfo` menu {kebab} and choose *Delete Project*.
 
+. Type `bookinfo` in the confirmation dialog box and click *Delete*.
++
+** Alternatively, you can run this command from the CLI to create the `bookinfo` project.
++
 ----
-$ ./cleanup.sh
-namespace ? [default] myproject
+$ oc delete project bookinfo
 ----
 
-. Confirm shutdown by running these commands:
-+
+[id="ossm-remove-bookinfo-smmr_{context}"]
+== Remove the bookinfo project from the {ProductShortName} member roll
+
+.Procedure
 
+. Log into the {product-title} web console.
+
+. Click *Catalog* -> *Installed Operators*.
+
+. Click the *Project* menu and choose `openshift-operators` from the list.
+
+. Click the *Istio Service Mesh Member Roll* link under *Provided APIS* for the *{ProductName}* Operator.
+
+. Click the `ServiceMeshMemberRoll` menu {kebab} and select *Edit Service Mesh Member Roll*.
+
+. Edit the default Service Mesh Member Roll yaml and remove `bookinfo` from the *members* list.
++
+** Alternatively, you can run this command from the CLI to remove the `bookinfo` project from the `ServiceMeshMemberRoll`. Replace `<control plane project>` with the name of your control plane project.
++
 ----
-$ oc get virtualservices -n myproject
-No resources found.
-$ oc get gateway -n myproject
-No resources found.
-$ oc get pods -n myproject
-No resources found.
+$ oc -n <control plane project> patch --type='json' smmr default -p '[{"op": "remove", "path": "/spec/members", "value":["'"${bookinfo}"'"]}]'
```

Comment:
This looks incorrect
"value":["'"${bookinfo}"'"]

Maybe the ${} should be removed?  So it looks like this instead:
"value":["'"bookinfo"'"]

Reply:
Fixed.